mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/store/JobArtifact.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.server.master.store;
+
+import io.mantisrx.shaded.com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.mantisrx.shaded.com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Instant;
+import java.util.Map;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class JobArtifact {
+    @JsonProperty("artifactID")
+    ArtifactID artifactID;
+    @JsonProperty("project")
+    String project;
+    @JsonProperty("version")
+    String version;
+    @JsonProperty("createdAt")
+    Instant createdAt;
+    @JsonProperty("runtimeType")
+    String runtimeType;
+    @JsonProperty("dependencies")
+    Map<String, String> dependencies;
+    @JsonProperty("entrypoint")
+    String entrypoint;
```

Comment:
is this needed?

Reply:
you mean the `entrypoint`? good question. I foresee TEs in the future to only use metadata from DB to instantiate a job, so I guess the entrypoint is important?

Reply:
What do you think?

Reply:
The current structure has some metadata files inside job jar to mark runtime type and 'entry class'.

Reply:
yeah I'm thinking about getting rid of that file in the future. this gets us one step closer!